### ISSUE TITLE
SafariStand doesn't work on El Capitan 10.11.6, Safari 11

### ISSUE THREAD

**Issue by sxandy** [anakinsk/SafariStand#52]
It doesn't work at all.

**Comment by JayBrown** [comment]
Already mentioned [**here**](https://github.com/anakinsk/SafariStand/issues/51#issuecomment-330712404).

**Comment by anakinsk** [comment]
This has a lower priority then https://github.com/anakinsk/SafariStand/issues/51.
Chances are this will be fixed with introducing new method of fetching favicons.  If not, planned for release 11.0.2

**Comment by kinoute** [comment]
just a reminder, binary files for El Capitan 10.11.6 Safari 11 available here for your patcher:

https://github.com/anakinsk/SafariStand/issues/51#issuecomment-330986618

**Comment by anakinsk** [comment]
@kinoute yep thanks a lot for that, I remember. El Capitan patcher is the first next thing on my to do list, I just need to setup appropriate virtual machine environment to test it properly.

Thanks for your contribution!
